### PR TITLE
add wc3 exe paths from 1.32

### PR DESCRIPTION
**Pull request by Cokemonkey11**
and upgrade gradle wrapper + use new toolchain management

### CODE REVIEW

**Comment by Frotty** [comment]
Meh, I thought PRs are checked by circleCI, but turns out that's just from codebeat 🤦‍♂️ 
build fails https://app.circleci.com/pipelines/github/inwc3/wc3libs/47/workflows/fb23d15d-025c-47cf-94c1-238fb59eec63/jobs/50

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-    id "java"
+    id "java-library"
     id "antlr"
     id "jacoco"
     id "idea"
 }
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+java {
```
Comment (Frotty) [inline build.gradle:8]: doesn't work

**Comment by Frotty** [comment]
was just gonna say, why do you mix feature with gradle stuff again...

**Comment by Cokemonkey11** [comment]
I did gradlew build check test locally and it was all green. Will check the
CI later today

Slightly better question is: why are CI checks inconsistent with build
result?

**Comment by Frotty** [comment]
I reset and pushed the path additions separetly.

**Pull request merged**